### DISCUSSION TITLE
Container stopped, cannot restart

### DISCUSSION THREAD

**Discussion by memilanuk** [tteck/Proxmox#2556] [Help Needed] [answered]
So... I noticed that my sonarr LXC was complaining that the download client (sabnzbd) was unavailable due to failure.  Couldn't load the web interface for the sabnzbd LXC container.  Went into proxmox, and the container is marked as stopped/locked:

![image](https://github.com/tteck/Proxmox/assets/1961428/d240fa5f-44ca-4bb5-825c-3f2d5d3bf4da)

and I can't get a status, open a console on it, nothing.  It defaults to the snapshots page for that container, which looks like this now:

![image](https://github.com/tteck/Proxmox/assets/1961428/b49c4e87-2d15-43b2-bd46-dd69a515986c)

So... now I'm kinda stuck.  Can't update the container, can't get into the container, can't seem to do *anything* with the container.   Can't roll back the snapshot, as I get this error:

![image](https://github.com/tteck/Proxmox/assets/1961428/6f285a59-b77b-40f9-bfc9-540965a7fc45)

FWIW, `/mnt/nas-data` is the mount point to my Synology NAS:

![image](https://github.com/tteck/Proxmox/assets/1961428/f9a0a26f-c7ed-4061-911d-151a74158d0f)

Any suggestions, short of re-creating the container from scratch?

**Comment by tteck** [comment]
container storage full?

**Reply by tteck** [reply, marked as answer]
`pct unlock 108`

**Reply by memilanuk** [reply]
> container storage full?

Synology DSM shows 13+TB available still...

> `pct unlock 108`

That seems to have done the job... the container is back running and available to sonarr.  

Any guesses as to why it got locked?
